packages/docs/docs/auth/smart-scopes.md

```diff
@@ -0,0 +1,31 @@
+# SMART Scopes for Access Control
+
+SMART on FHIR’s authorization scheme uses OAuth scopes to communicate (and negotiate) access requirements. There is relevant [background information on HL7.org](http://hl7.org/fhir/smart-app-launch/STU2/scopes-and-launch-context.html).
```

Comment:
Maybe link the words "Oauth scopes" to some basic supporting info about what Oauth scopes are? I found the Auth0 docs pretty well written: https://auth0.com/docs/get-started/apis/scopes